doc/SdA.txt

```diff
@@ -80,20 +83,20 @@ representations of intermediate layers of the MLP.
 ``self.sigmoid_layers`` will store the sigmoid layers of the MLP facade, while
 ``self.dA_layers`` will store  the denoising autoencoder associated with the layers of the MLP. 
 
-Next step, we construct ``n_layers`` sigmoid layers (we use the
-``HiddenLayer`` class introduced in :ref:`mlp`, with the only
-modification that we replaced the non-linearity from ``tanh`` to the
-logistic function :math:`s(x) = \frac{1}{1+e^{-x}}`) and ``n_layers``
-denoising autoencoders, where ``n_layers`` is the depth of our model.
-We link the sigmoid layers such that they form an MLP, and construct
-each denoising autoencoder such that they share the weight matrix and the 
-bias of the encoding part with its corresponding sigmoid layer.
+Next, we construct ``n_layers`` denoising autoencoders and ``n_layers`` sigmoid
```

Comment:
We construct the sigmoid layers first, so I would not say first that we construct autoencoders.